ui/app/AppLayouts/Wallet/controls/DappsComboBox.qml

```diff
@@ -26,7 +26,7 @@ ComboBox {
 
     background: SQP.StatusComboboxBackground {
         objectName: "dappsBackground"
-        active: root.down || root.hovered
+        active: root.down || root.hovered || !root.enabled
```

Comment:
Is this intended?

Reply:
It was needed for disabling the DApps button. But after discussions with UX team, DApps button will be either enabled or invisible.
Removing this change